crates/biome_js_analyze/src/lint/nursery/no_playwright_valid_describe_callback.rs

```diff
@@ -0,0 +1,164 @@
+                        let name = id.name().ok()?;
+                        let token = name.value_token().ok()?;
+                        token.text_trimmed() == "test"
+                    } else {
+                        false
+                    }
+                } else {
+                    false
+                }
+            }
+            _ => false,
+        };
+
+        if !is_describe {
+            return None;
+        }
+
+        // Get the callback argument (should be the second argument for describe calls)
+        let args = call_expr.arguments().ok()?;
+        let [_, Some(callback_arg)] = args.get_arguments_by_index([0, 1]) else {
+            return None;
+        };
+        let callback_expr = callback_arg.as_any_js_expression()?;
+
+        // Check if it's a function
+        match callback_expr {
+            AnyJsExpression::JsArrowFunctionExpression(arrow) => {
+                // Check if async
+                if arrow.async_token().is_some() {
+                    return Some(InvalidReason::Async);
+                }
+
+                // Check if has parameters
+                if let Ok(params) = arrow.parameters() {
+                    let has_params = match params {
+                        biome_js_syntax::AnyJsArrowFunctionParameters::AnyJsBinding(_) => true,
+                        biome_js_syntax::AnyJsArrowFunctionParameters::JsParameters(p) => {
+                            p.items().len() > 0
+                        }
+                    };
+                    if has_params {
+                        return Some(InvalidReason::HasParameters);
+                    }
+                }
+            }
+            AnyJsExpression::JsFunctionExpression(func) => {
+                // Check if async
+                if func.async_token().is_some() {
+                    return Some(InvalidReason::Async);
+                }
+
+                // Check if has parameters
+                if let Ok(params) = func.parameters() {
+                    if params.items().len() > 0 {
+                        return Some(InvalidReason::HasParameters);
+                    }
+                }
+            }
+            _ => return None, // Not a function, but we won't report this
+        }
+
+        None
+    }
+
+    fn diagnostic(ctx: &RuleContext<Self>, state: &Self::State) -> Option<RuleDiagnostic> {
+        let node = ctx.query();
+
+        let (message, note) = match state {
+            InvalidReason::Async => (
+                markup! { "Describe callback should not be "<Emphasis>"async"</Emphasis>"." },
+                markup! { "Remove the "<Emphasis>"async"</Emphasis>" keyword from the describe callback." },
+            ),
+            InvalidReason::HasParameters => (
+                markup! { "Describe callback should not have parameters." },
+                markup! { "Remove parameters from the describe callback." },
+            ),
+        };
```

Comment:
This diagnostic has the "what", and "how to fix", but doesn't explain _why_ describe blocks shouldn't have `async`.